src/hooks/useConceptExerciseIssues.ts

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react'
+
+export interface ConceptExerciseIssue {
+  number: number
+  title: string
+  body: string
+  url: string
+  updatedAt: string
+}
+
+export type ConceptExerciseIssueResult<T> = {
+  result: T | undefined
+  error: boolean
+  done: boolean
+}
+
+function useConceptExerciseIssuesApi<T>(
+  url: string
+): ConceptExerciseIssueResult<T> {
+  const [state, setState] = useState<ConceptExerciseIssueResult<T>>({
+    result: undefined,
+    error: false,
+    done: false,
+  })
+
+  useEffect(() => {
+    if (state.done === true) {
+      return
+    }
+
+    let isMounted = true
+
+    fetch(url)
```

Comment:
If you want JSON, add `, headers: { accept: 'application/json' }`. Sure, it defaults to JSON, but it doesn't need to.